_data/current.yml

```diff
@@ -302,3 +302,9 @@
   url: https://agiletestingdays.com/?utm_source=testingconferences
   twitter: AgileTD
   status: Super Early Bird Registration is Open until April 27, 2025
+
+- name: "Workshop: Everyone wants to shift left, but our tests are just too slow!"
+  location: Online
+  dates: "June 25, 2025"
```

Comment:
Please put this workshop in the right place in the list. Move it up so it is in order by date. Thanks